src/experiment/experiment.js

```diff
@@ -36,13 +37,9 @@ export const preloadObj2contentObj = (preloadObj) => {
   }, {});
 };
 
-// Automatically populate the audioContent object with the audio files
-// Copied audioContent from preload.js
-const audioBlocks = {
-  3: [mediaAssets.audio["feedbackCorrect"], mediaAssets.audio["feedbackIncorrect"]],
-};
-
-export const audioContent = preloadObj2contentObj(audioBlocks);
+const bucketURI = 'https://storage.googleapis.com/roar-mp' // TODO: separate buckets into "spanish" and "english" variants
+export let mediaAssets = generateAssetObject(assets, bucketURI)
+export let preloadTrials = createPreloadTrials(assets, bucketURI).default
 
 
 export function buildExperiment() {
```

Comment:
First lines of buildExperiment we need to call  initRoarJsPsych(config) and initiate the timeline with initRoarTimeline(config). roar timeline should be set to the variable (timeline, initialTimeline) where the rest of the experiments are pushed to.